### ISSUE TITLE
aws_wafv2_web_acl: managed-rule-group-statement is missing Version option

### ISSUE THREAD

**Issue by davy-oo** [hashicorp/terraform-provider-aws#21546]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS WAF offers an optional "Version" to stick to a static version for ManagedRuleGroupStatement. See [AWS APIReference](https://docs.aws.amazon.com/waf/latest/APIReference/API_ManagedRuleGroupStatement.html).

The AWS provider does not seem to support it according to the [latest 3.60 doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#managed-rule-group-statement).

The version option is useful to test new set of coming rules. AWS released this new option on Aug 9, 2021: https://aws.amazon.com/about-aws/whats-new/2021/08/aws-waf-offers-managed-rule-group-versioning/

Use case: AWS sent recommendation like this one by email:
_"On October 27, 2021, we published a new version of the Core Rule Set, currently available as a static version, named Version_1.2, in the console. On November 3, 2021. the Default version for this rule group will be updated to the same configuration as Version_1.2. If you are using the Default version, Version_1.2 can be tested prior to November 3, 2021 by switching to the static version.If you require more time to test the new version we recommend staying on or moving to Version_1.1(previous version) before November 3, 2021. "_

### New or Affected Resource(s)

* aws_wafv2_web_acl, managed-rule-group-statement

### Potential Terraform Configuration

See potential new "version" option below.

```hcl
resource "aws_wafv2_web_acl" "example" {
...
statement {
      managed_rule_group_statement {
        name             = "AWSManagedRulesCommonRuleSet"
        version          = "Version_1.1"
        vendor_name      = "AWS"
        excluded_rule {
          name = "SizeRestrictions_BODY"
        }
      }
    }
...
}
```

### References
* https://docs.aws.amazon.com/waf/latest/APIReference/API_ManagedRuleGroupStatement.html
* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#managed-rule-group-statement
* https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-changelog.html

**Comment by galoitsu** [comment]
I have worked on this issue [here](https://github.com/hashicorp/terraform-provider-aws/pull/21732).

**Comment by breathingdust** [comment]
Hi all 👋  Just letting you know that this is issue is featured on this quarters [roadmap](https://github.com/hashicorp/terraform-provider-aws/blob/main/ROADMAP.md). If a PR exists to close the issue a maintainer will review and either make changes directly, or work with the original author to get the contribution merged. If you have written a PR to resolve the issue please ensure the "Allow edits from maintainers" box is checked. Thanks for your patience and we are looking forward to getting this merged soon!

**Comment by YakDriver** [comment]
Closed by #21732
